drake/CMakeLists.txt

```diff
@@ -108,6 +108,15 @@ mex_setup()
 #   message("Setting CTEST Configuration Type to ${CTEST_CONFIGURATION_TYPE}")
 #endif()
 
+file(WRITE ${PROJECT_SOURCE_DIR}/core/Path.h
```

Comment:
Not necessary for now, but would it be an idea to put this in the PROJECT_BINARY_DIR instead of PROJECT_SOURCE_DIR, since it's a build product? That's what CMake's GenerateExportHeader does by default as well.